be/src/storage/lake/delta_writer.cpp

```diff
@@ -171,11 +187,13 @@ inline Status DeltaWriterImpl::flush() {
 // in a bthread.
 Status DeltaWriterImpl::open() {
     SCOPED_THREAD_LOCAL_MEM_SETTER(_mem_tracker, false);
+    RETURN_IF_ERROR(build_schema());
```

Comment:
Cannot call `build_schema()` here because `build_schema()` will call `tablet.get_schema()` and `tablet.get_schema()` may trigger network access. As mentioned in the comment, do not perform any I/O in this method.

Reply:
Done.